.yamllint

```diff
@@ -1,6 +1,10 @@
 ---
 extends: default
 rules:
+  document-start: disable
   line-length:
     max: 200
     level: warning
+  indentation:
+    spaces: 2
+    indent-sequences: false
```

Comment:
this will touch almost all files and break with the formatting we were using before.
do you really want to change this in all roles now?

Reply:
Basically, yes.
The reason is for compatibility with https://github.com/macisamuele/language-formatters-pre-commit-hooks `pretty-format-yaml`, i.e. you should have to worry about the formatting it should be fixed automatically.
I couldn't see any other YAML formatter hooks that could adhere exactly to the yamllint defaults?

Reply:
I see, if that is needed for getting autoformatting, that's a good enough reason for me.
Thanks for clarifying